server/initPassport.js

```diff
@@ -1,18 +1,20 @@
 const passport = require('passport');
 const FacebookStrategy = require('passport-facebook').Strategy;
+const { checkOrCreateUser } = require('./utilities/utilityFunctions');
 require('dotenv').config();
 
 passport.use(new FacebookStrategy({
-    clientID: process.env.FB_APPID || '1871157256463548',
-    clientSecret: process.env.FB_APPSECRET,
-    callbackURL: 'http://www.google.com'
+  clientID: process.env.FB_APPID || '1871157256463548',
+  clientSecret: process.env.FB_APPSECRET,
+  callbackURL: 'localhost:3000/'
```

Comment:
Could you change this to tailbud.herokuapp.com or try some form of process.env.url?